File: Studio.UI/src/app/data/results/locations/LocationResult.ts
import { Guid } from "guid-typescript";
import { BaseResult } from "../base/BaseResult";
import { EventResult } from "../events/EventResult";
import { CityResult } from "./CityResult";

export interface LocationResult extends BaseResult{
    locationName: string;
    cityId: Guid;
    city: CityResult;
    events: EventResult[];
}

File: Studio.API/Studio.API/Business/Domain/Results/Locations/LocationResult.cs

using Studio.API.Business.Domain.Results.Bases;
using Studio.API.Business.Domain.Results.Events;
using System.ComponentModel.DataAnnotations.Schema;

namespace Studio.API.Business.Domain.Results.Locations
{
    public class LocationResult : BaseResult
    {
        public string LocationName { get; set; }
        
        public Guid CityId { get; set; }
        
        public CityResult City { get; set; }
        public IList<EventResult> Events { get; set; }
    }
}
